app/js/controllers/home.js

```diff
@@ -1,37 +1,47 @@
 'use strict';
+const HomeCtrl = ($scope, $state, ObservationService, SchoolService) => {
 
-import moment from 'moment';
+    let observationToBeDeleted;
 
-const HomeCtrl = ($scope, $rootScope, $state) => {
+    // fetch data
+    ObservationService.fetchObservations((err, res) => {
+        if (!err) {
+            $scope.observations = res.data.data;
+        } else {
+            console.error(err, 'ERROR');
+        }
+    });
 
-    $scope.page = 'observed';
+    SchoolService.fetchSchools((err, res) => {
+        if (err) {
+            console.error(err);
+        }
+        $scope.schools = res.data.data;
+    });
 
-    $scope.observations = [{
-        observationKind: 'Lesson',
-        readableDate: '11/13/2017',
-        grade: 'Kindergarten',
-        school: 'UPSM ELEMENTARY',
-        teacher: 'Mr.Martin'
-    }, {
-        observationKind: 'Crew',
-        readableDate: '11/13/2017',
-        school: 'ELLEN THOMPSON ELEMENTARY',
-        teacher: 'Ms.Andrews'
-    }, {
-        observationKind: 'Lesson',
-        readableDate: '11/13/2017',
-        school: 'UPSM ELEMENTARY',
-        subject: 'English',
-        teacher: 'Ms.Andrews'
-    }, {
-        grade: '1st grade',
-        observationKind: 'Lesson',
-        readableDate: '11/13/2017',
-        school: 'ELLEN THOMPSON ELEMENTARY',
-        subject: 'Science',
-        teacher: 'Ms.Andrews'
-    }];
+    $scope.openModal = (observation) => {
+        angular.element(document.getElementsByClassName('delete-observation-modal')).css('display', 'flex');
+        observationToBeDeleted = observation;
+    };
+
+    $scope.closeModal = () => {
+        angular.element(document.getElementsByClassName('delete-observation-modal')).css('display', 'none');
+    };
+
+    $scope.deleteObservation = () => {
+        ObservationService.remove({
+            id: observationToBeDeleted.id
+        }, (res) => {
+            angular.element(document.getElementsByClassName('delete-observation-modal')).css('display', 'none');
+            $state.reload();
```

Comment:
@teddyotieno Rather than reload could we not just remove the delete observation from observations variable and that should remove it from the UI list ?

Reply:
Fixed the issue 👍